### PR TITLE
#4796 [Bug] Should delete related files when delete workspace

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4796 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by wu87988622** [comment]
retry

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/Workspace/Settings/Settings.js [hunk 1]
```diff
@@ -81,6 +82,7 @@ const Settings = () => {
         zookeeper: convertIdToKey(zookeeperId),
         broker: convertIdToKey(brokerId),
         worker: convertIdToKey(workerId),
+        files,
```
Comment (eechih) [inline ohara-manager/client/src/components/Workspace/Settings/Settings.js:85]: 建議把 workspace, zookeeper, broker, worker, files 封裝在
 deleteWorkspaceHooks.useDeleteWorkspaceAction 內。

Comment (saivirtue) [reply ohara-manager/client/src/components/Workspace/Settings/Settings.js:85]: 這隻PR應該是只處理delete files部分；refacor在#4891有處理了 :)

Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/Settings.js:85]: Got it

**Comment by wu87988622** [comment]
@eechih This pr is ready to merge.
Thanks for @saivirtue help.

**Pull request closed without merging**